package.json

```diff
@@ -25,29 +25,25 @@
   },
   "private": true,
   "peerDependencies": {
-    "@angular/animations": "^11.2.4",
     "@angular/cdk": "^11.2.4",
-    "@angular/common": "^11.2.4",
-    "@angular/core": "^11.2.4",
-    "@angular/forms": "^11.2.4",
     "@fortawesome/fontawesome-pro": "^5.7.2",
     "d3": "^5.7.0",
     "ngx-bootstrap": "^6.2.0"
```

Comment:
According to https://github.com/valor-software/ngx-bootstrap#compatibility we can and probably should move to v7 - but you should see what changes are in v7 to see if there's anything we need to update. This comment applies to Spike also. If you do update you'll need to update this reference in peerDependencies and the other in devDependencies